components/esp_modem/include/cxx_include/esp_modem_cmux.hpp

```diff
@@ -84,6 +91,9 @@ class CMux {
     static uint8_t fcs_crc(const uint8_t frame[6]);     /*!< Utility to calculate FCS CRC */
     void data_available(uint8_t *data, size_t len);     /*!< Called when valid data available */
     void send_sabm(size_t i);                           /*!< Sending initial SABM */
+    void send_disc(size_t i);                           /*!< Sending closing request for each virtual terminal */
```

Comment:
I'd rather keep the *disc* name here, as we already have a *close* command and they differ:
* `send_disc()` closes **one** virtual terminal
* `close()` exits the control mode of CMUX, so closes the entire mode.

Suggest renaming to `send_disconnect()`, WYT?

Reply:
Actually, `send_disconnect` was my first idea, but I saw the comment on sending closing. 

Maybe rename `close` to `exit_cmux`or `close_all`

Reply:
I wen with `send_disconnect()` and made a special case for closing the *control* terminal as `send_disconnect(0)` (since the control terminal is referred as terminal 0.